tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+requires =
+	tox>=4
+env_list = lint, integration
+
+[testenv]
+setenv =
+	version=3.6.4
+	repo=https://github.com/canonical/zookeeper-k8s-operator.git
+	base-name=docker.io/dataplatformoci/zookeeper
+
+[testenv:lint]
+description = run linters
+skip_install = true
+deps =
+	yamllint
+commands = 
+	yamllint --no-warnings rockcraft.yaml .github
+
+[testenv:integration]
+description = run operator integration tests
+skip_install = true
+allowlist_externals =
+	tox
+	microk8s
+	bash
+commands = 
+	bash -ec 'if ! [ -f zookeeper_{env:version}_amd64.rock ]; then rockcraft pack; fi' {posargs}
+	microk8s ctr image import zookeeper_{env:version}_amd64.rock --base-name {env:base-name}
+	bash -ec 'if ! [ -d operator ]; then git clone {env:repo} operator ; fi' {posargs}
```

Comment:
This should have targets for each integration tox environment and be called individually through github actions for test concurrency.

Reply:
This was done in purpose to avoid inconsistencies of the tests run between the charm and the test run on when we build a new image. It is a tradeoff discussed with Enrico and he agreed on that point

Reply:
If thats the conclusion that yall came to then I trust your judgment :)